### ISSUE TITLE
Question regarding MyApi 

### ISSUE THREAD

**Issue by testertest089** [GoogleCloudPlatform/gradle-appengine-templates#36]
I can't run the application because of an error regarding not being able to find the symbol class MyApi. I don't understand the MyApi usage in the EndpointsAsyncTask class. If it is a dummy class than is it referring to the API name defined in MyEndpoints? I was unable to find a MyApi class in the reference section of the android developer website. My libraries and plug-ins are up to date. I'm new to developing backend APIs so I apologize for the inconvenience. Thank you.

**Comment by loosebazooka** [comment]
I have a feeling that the build configuration might be off.  Can you check you "app" build.gradle file to see if you see the line

```
compile project(path: '<backend-moduleName>', configuration: 'android-endpoints')
```

**Comment by testertest089** [comment]
The configuration was the problem. The error was strange but the configuration wasn’t initially being created and then the gradle sync wasn’t recognizing the compile project command when I entered it manually. I kept running it and restarted Android Studio and finally it recognized the compile project command. Thank you. 

Also, just for accuracy, the syntax I used was: (path: ':backend', configuration: 'android-endpoints') 
I was getting a “Project with path 'backend' could not be found in project ':app'.”  error with the syntax recommended.
